packages/eslint-plugin-next/lib/rules/image-component/no-absolute-paths.js

```diff
@@ -0,0 +1,30 @@
+const createImageRule = require('../../utils/imageComponentRule.js')
+const NodeAttributes = require('../../utils/nodeAttributes.js')
+
+module.exports = {
+  meta: {
+    messages: {
+      noAbsolutePaths:
+        'You are using an absolute path in the src attribute of the next/image component.' +
+        'This is almost definitely a mistake--use the "unoptimized" attribute to use ' +
```

Comment:
adding "unoptimized" is a last resort opt out right?
point to documentation link to what they should actually do to optimize?